### PR TITLE
feat: clear script button

### LINKED ISSUE #5005: New Query/Script Option
## User Story

<p></p>

As a user, I want to be able to start a brand new query/script so that I can start something new, even if I was working on a different query/script.

<p></p>

## Acceptance Criteria

<p></p>

- An option will be available to start a new script/query

- When selected, if a query is in draft state, the user will be given the option to discard or save the script/query they are working on

- When selected, any bucket, measurement, tag, or field selection in the schema browser will be removed and the user will be back at bucket selection in that browser

- When selected, the flux editing window will be reset to blank

- When selected, any table or graph results that were previously generated will be wiped out

- When starting a brand new session, and navigating to the Data Explorer, he user will be shown an option to start a new query/script

- When navigating away from and back to the data explorer within the same session, the user will be shown the data explorer state as it was last in before they left it (different from first use in new session)

- New query/script will have the name “New Query/Script (Draft)”

- The user will be able to rename this new query/script something else

- Upon rename, the query will be saved (see save user story)

<p></p>

## Notes

<p></p>

Reference:  Proposal on behavior:  <https://docs.google.com/document/d/13C6JZ7CscZMTIzG_3WMCgnOCCjtZYkGKNDUgSh0YiPw/edit#heading=h.mm433mqts43v>

### LINKED ISSUE #5086: Add "new" button in flux query builder
Add a "new" button in Flux Query Builder that will reset the query editor and return the user into editing a draft file.

### PR DESCRIPTION
**Pull request by drdelambre**
Closes #5005 
Closes #5086 

adds a "new script" button to clear out the:
query text
download status
results

Kapture 2022-07-25 at 18 22 34

### CODE REVIEW

**Review by appletreeisyellow — COMMENTED** [review]
Should we clear schema browser in this PR?

**Comment by drdelambre** [comment]
i was on the fence about that to. i'm unclear on what a "new query" entails. timerange too? lets bring it up with product!

**Comment by drdelambre** [comment]
just saw the schema browser condition in the ticket. ADDED!

File: src/dataExplorer/context/persistance.tsx [hunk 1]
```diff
@@ -2,29 +2,42 @@ import React, {FC, createContext} from 'react'
...
 }
 
 const DEFAULT_CONTEXT = {
   horizontal: [0.2],
   vertical: [0.25, 0.8],
   range: DEFAULT_TIME_RANGE,
   query: '',
+  selection: {
+    bucket: null,
+    measurement: null,
```
Comment (appletreeisyellow) [inline src/dataExplorer/context/persistance.tsx:33]: Should `measurement` be an empty string `''`?

**Review by appletreeisyellow — APPROVED** [review]
I like the `persistance` context idea that consolidates and manages all the session storage! Added a comment but not a blocker

**Pull request merged**